pkg/uc/dependencymanager/container_test_types.go

```diff
@@ -21,4 +21,5 @@ type ContainerRunConfig struct {
 	ExtraHosts   []string                             `yaml:"extra_hosts"`
 	Labels       map[string]string                    `yaml:"labels"`
 	PortBindings map[docker.Port][]docker.PortBinding `yaml:"port_bindings"`
+	Auth         docker.AuthConfiguration             `json:"dockerauth"`
```

Comment:
yaml: instead of json

Reply:
Sure, I didn't push the change because I'm testing in local.